kernel/src/actions/visitors.rs

```diff
@@ -259,10 +260,10 @@ impl TransactionVisitor {
         row_index: usize,
         app_id: String,
         getters: &[&'a dyn GetData<'a>],
-    ) -> DeltaResult<Transaction> {
+    ) -> DeltaResult<SetTransaction> {
```

Comment:
(comment here because github sucks)

The struct should now be called `SetTransactionVisitor`

Reply:
yep done, and did a few more renames just to prefix everything with `Set`